front/app/resources/GetPhase.tsx

```diff
@@ -62,7 +64,9 @@ export default class GetPhase extends React.Component<Props, State> {
   }
 
   componentWillUnmount() {
-    this.subscriptions.forEach((subscription) => subscription.unsubscribe());
+    if (this.subscriptions) {
```

Comment:
Do we need this since it is an array? The forEach should capture it

Reply:
@EdwinKato Actually - I see an error when I don't have this in and try to load the new idea form page:

![image](https://user-images.githubusercontent.com/33987955/196368097-9054079d-41ab-411a-b9ea-1287fcf1936d.png)

Reply:
Maybe there is something else going on then?

Reply:
Ahh I see.. it looks like it happens when there is no active phase in a timeline project (even when I've set subscriptions to an empty array at the start) and I view the project page.

Reply:
Oh nevermind - it is a different error I'm seeing actually. In getProjectId instead.